src/lib/data/realtime.ts

```diff
@@ -72,7 +72,7 @@ export async function onLocation(
 
   return db.write(async () => {
     return realtimeData.update(record => {
-      record.speed = speed;
+      record.speed = speed ? speed * 3.6 : 0; // Convert m/s to km/h as thats more useful.
```

Comment:
@craigmulligan is this the right place to do the conversion of speed from m/s to km/h?

Reply:
That works, do you think we would ever need meters per second? If we do need meters per second somewhere else then I'd just keep this as metres and have a util function that converts it on render in the component.

Reply:
No I can't see a use for m/s for now and I think having km/h as the base speed for everything else is probably best as I think this is what we would want to write into file, display, etc.